config/parse_vault_mount.go

```diff
@@ -16,7 +15,7 @@ func (c *Config) processVaultMounts(list *ast.ObjectList, environment *Environme
 	for _, mountAST := range list.Items {
 		x := mountAST.Val.(*ast.ObjectType).List
 
-		valid := []string{"config", "role", "type", "path"}
+		valid := []string{"config", "role", "type", "path", "maxleasettl"}
```

Comment:
ok, please do that, and it will be much better to update the example to include these  as well.
thanks!